spec/controllers/books_controller_spec.rb

```diff
@@ -1,4 +1,43 @@
 require 'rails_helper'
+require 'support/shared_context'
 
 RSpec.describe BooksController, type: :controller do
```

Comment:
Remember what we talked about `Rspec.` and the `type: :controller`. We don't need them!